packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -539,7 +539,7 @@ async function getUmaPriceAtTimestamp(timestamp) {
     return ethExchangeRate;
   } catch (err) {
     console.error("Failed to fetch UMA historical price from Coingecko, falling back to default");
-    return 0.1;
+    return 0.0016;
```

Comment:
Does this happen often when running the script?

Reply:
a non zero amount of timestamps queried when running the script it happened 32 times.